src/bentoml_cli/containerize.py

```diff
@@ -1,360 +1,636 @@
 from __future__ import annotations
 
+import os
```

Comment:
This file mainly contains backward compatibility layer for our current containerize behaviour.